astrowidgets/_astropy_init.py

```diff
@@ -50,8 +50,8 @@
                 try:
                     update_default_config(__package__, config_dir)
                 except ConfigurationDefaultMissingError as e:
-                    wmsg = (e.args[0] +
-                            " Cannot install default profile. If you are "
+                    wmsg = (e.args[0] + " Cannot"
```

Comment:
p.s. I never completely agree with this particular PEP 8 check. I am open to adding stuff to be ignored for `[flake8]` if they become too cumbersome.

Reply:
this seems fine -- I was really onfused last night for a while because I needed to rebase to see what lines were causing the issue.